src/components/Personalization/index.js

```diff
@@ -44,46 +79,59 @@ const createCollect = eventManager => {
   };
 };
 
+const validateOptions = options => {
+  const validate = objectOf(GET_DECISIONS_OPTIONS_SCHEMA);
+  const { viewStart, scopes } = validate(options);
+
+  if (viewStart == null && scopes.length === 0) {
```

Comment:
This needs to change too right, since `viewStart` won't ever be `null`?

Reply:
this is why I needed to make a difference when `viewStart` is missing. If `viewStart` is missing and `scopes` is empty or missing as well then we should trigger an error. OR we return always an empty array and we never throw an error. I would return an empty array rather than throw an error, what do you think?

Reply:
I think I don't really understand how viewStart is involved with scopes. Maybe you can provide some context here?

Reply:
And could someone want to get all scopes?

Reply:
We might have decisions for page-wide scope, those decisions don't have a scope parameter. Or we might have specific scopes decisions.
There will be use cases when the customer will want to getDecisions for "Foo1" scope, or for a page-wide scope, or for both "page-wide" and "Foo1". 
We talked about the use case when the customer will want to get everything he has in the decision storage and we are not going to support it now.

Reply:
`viewStart` is a flag for page-wide scope decisions